### ISSUE TITLE
windows build issue

### ISSUE THREAD

**Issue by selvam920** [pichillilorenzo/flutter_inappwebview#2325]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

F:\Project\cable\cable_app\windows\flutter\ephemeral\.plugin_symlinks\flutter_inappwebview_windows\windows\types\web_resource_response.cpp(54,28): warning C4244: 'argument': conversion from '__int64' to 'int', possible loss of data [F:\Project\cable\cable_app\build\windows\x64\plugins\flutter_inappwebview_windows\flutter_inappwebview_windows_plugin.vcxproj]
F:\Project\cable\cable_app\build\windows\x64\packages\Microsoft.Web.WebView2\build\native\include\WebView2EnvironmentOptions.h(194,3): warning C4458: declaration of 'value' hides class member [F:\Project\cable\cable_app\build\windows\x64\plugins\flutter_inappwebview_windows\flutter_inappwebview_windows_plugin.vcxproj]
F:\Project\cable\cable_app\build\windows\x64\packages\Microsoft.Web.WebView2\build\native\include\WebView2EnvironmentOptions.h(193,3): warning C4458: declaration of 'value' hides class member [F:\Project\cable\cable_app\build\windows\x64\plugins\flutter_inappwebview_windows\flutter_inappwebview_windows_plugin.vcxproj]

### Expected Behavior

should be build

### Steps with code example to reproduce

<details>
<summary>Steps with code example to reproduce</summary>

```dart
// Paste your code here
```
</details>

### Stacktrace/Logs

logs attached

### Flutter version

latest stable

### Operating System, Device-specific and/or Tool

windows 11

### Plugin version

latest

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by pichillilorenzo** [comment]
They are warnings, you should be able to build

**Comment by selvam920** [comment]
yes, after got the warinng build succeed, can it remove the warnings?

**Comment by pichillilorenzo** [comment]
Unfortunately a couple of them are coming from official WebView2 windows library, so nope.
I have already tried, but it seems not possible as they are coming from the dependency.

**Comment by ShareZore** [comment]
I also have this warning, can compile and pack normally. However, the application cannot be opened after installation packaged as.exe.
